draft-ietf-quic-qlog-h3-events.md

```diff
@@ -658,7 +673,20 @@ document as well.
 
 # IANA Considerations
 
-There are no IANA considerations.
+This document registers a new entry in the "qlog event extension schema
+identifer" registry.
+
+Extension URI:
```

Comment:
Not sure if this is actually an issue, but IIUC, this would mean that listing the H3 events in `additional_event_schemas` will lead to a URI like this:

`urn:ietf:params:qlog:h3#h3` 

Conceptually fine, but the duplication of h3 is a bit weird. Makes me wonder if in general we should go for the "full acronym" in the extension URI (i.e., http3 instead of h3, analogous to quic instead of q1)?

Reply:
yeah that makes sense

Reply:
how about `urn:ietf:params:qlog:http#h3`? That would make it easy to add support for other HTTP versions or extensions as new catagories

Reply:
Ah no my suggestion doesn't work because we don't allow category extensions

Reply:
Actually that's different. So its an open question, should we allow new categories to be added to a schema? 

In other words if the h3 spec registered `urn:ietf:params:qlog:http` and category `h3`, should we let other documents add categories? I think it is a reasonable thing to do but if so we would need to tweak the registration guidance.

Reply:
I gave my perspective on this in a separate comment below https://github.com/quicwg/qlog/pull/415#issuecomment-2191940230